<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fhead</class>
 <widget class="fhead" name="fhead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="headGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Head</string>
   </property>
   <widget class="QWidget" name="headGraph" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>IMAGE: key points &amp; current position</string>
     </property>
    </widget>
   </widget>
   <widget class="panControlWidget" name="panControl" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>650</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="tiltControlWidget" name="tiltControl" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>50</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRangePanTilt">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRangePanTilt_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pan : Tilt range</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="fixedPointButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>70</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Key point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>30</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="caprureButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>140</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="sliderGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>751</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Slider</string>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IMAGE: key points &amp; current position</string>
     </property>
    </widget>
   </widget>
   <widget class="sliderControlWidget" name="sliderControl" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>710</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="fpTableView">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>191</width>
     <height>441</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>47</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="focusUpButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>220</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Focus up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="focusDownButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>250</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Focus down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="liveViewButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>190</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>LiveView</string>
   </property>
  </widget>
  <widget class="QSlider" name="liveViewZoomSlider">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>panControlWidget</class>
   <extends>QWidget</extends>
   <header location="global">pancontrolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tiltControlWidget</class>
   <extends>QWidget</extends>
   <header location="global">tiltcontrolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sliderControlWidget</class>
   <extends>QWidget</extends>
   <header location="global">slidercontrolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>fhead</class>
   <extends>QWidget</extends>
   <header location="global">fhead.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
